pheweb/serve/server.py

```diff
@@ -224,7 +224,7 @@ def api_pheno(phenocode):
     if phenocode not in use_phenos:
         abort(404)
     try:
-        return jsonify( jeeves.get_pheno(phenocode))
+        return jeeves.get_pheno_manhattan(phenocode)
```

Comment:
This bit is only thing thats from your import branch..... seems that the new one just returns the json which works perfectly....

Reply:
I started the backend stuff from 3d1e686b1d13b3fc771e64414c02fb71b726dcab